hoot-core/src/main/cpp/hoot/core/conflate/ConflateExecutor.cpp

```diff
@@ -216,6 +218,23 @@ void ConflateExecutor::conflate(const QString& input1, const QString& input2, QS
 
   _load(input1, input2, map, isChangesetOutput);
 
+  // Check to see if all the data is untyped. If so, log a warning so the user knows they may not
+  // be getting the best conflate results in case types could be added to the input.
+  if (map->size() > 0 && !SchemaUtils::anyElementsHaveType(map))
+  {
```

Comment:
Moved this to here from `ConflateCmd`, so that the check happens if you're doing programmatic conflation via `ConflateExecutor` vs calling the `conflate` command.